### PR TITLE
Improve manual chapters

### PR DESCRIPTION
**Pull request by prckent**
Edits from Charlie Horak

### CODE REVIEW

File: manual/additional_tools.tex [hunk 1]
```diff
@@ -2,36 +2,36 @@ \chapter{Additional Tools}
...
 
-\section{Post-Processing}
+\section{Postprocessing}
   \subsection{qmca}
-    \ishell{qmca} is a versatile tool to analyze and plot the raw data from QMCPACK *.scalar.dat files.
-    It is a python executable and part of the Nexus suite of tools.  It can be found in 
-    \ishell{qmcpack/nexus/executables}. For more detail, see section~\ref{sec:qmca}.
+    \ishell{qmca} is a versatile tool to analyze and plot the raw data from QMCPACK \ishell{*.scalar.dat} files.
+    It is a Python executable and part of the Nexus suite of tools.  It can be found in 
+    \ishell{qmcpack/nexus/executables}. For details, see Section \ref{sec:qmca}.
```
Comment (ye-luo) [inline manual/additional_tools.tex]: `Section~\ref`

File: manual/additional_tools.tex [hunk 2]
```diff
@@ -2,36 +2,36 @@ \chapter{Additional Tools}
...
   \subsection{qmc-fit}
-    \ishell{qmc-fit} is a curve fitting tool to obtain statistical error bars on fitted parameters.
-    It is useful for DMC timestep extrapolation.  For more details, see section~\ref{sec:qmcfit}.
+    \ishell{qmc-fit} is a curve fitting tool used to obtain statistical error bars on fitted parameters.
+    It is useful for DMC time step extrapolation.  For details, see Section \ref{sec:qmcfit}.
```
Comment (ye-luo) [inline manual/additional_tools.tex]: `Section~\ref`

**Comment by prckent** [comment]
Updated to put a tilde in front of most \refs. A capture group in sed makes this simple:

sed -E 's/([a-z]) \\ref\{/\1~\\ref\{/g' -ibak *.tex

**Comment by prckent** [comment]
Test this please

**Pull request merged**